src/test/java/com/gary/backendv2/service/TrustPersonServiceTest.java

```diff
@@ -19,6 +22,41 @@ public class TrustPersonServiceTest {
 	private final UserRepository userRepository = mock(UserRepository.class);
 	private final TrustedPersonService trustedPersonService = new TrustedPersonService(trustedPersonRepository, userRepository);
 
+
+	@Test
+	void getAllTrustedPersons(){
+		List<TrustedPerson> expected = List.of(new TrustedPerson());
+
+		when(trustedPersonRepository.findAll()).thenReturn(expected);
+
+
+		var result = trustedPersonRepository.findAll();
+
+		assertEquals(expected.size(), result.size());
```

Comment:
you are not testing the service here
you mock a repository and then test whether that mock actually returns the value defined in `when` clause